### ISSUE TITLE
More general demo Binder repository? 

### ISSUE THREAD

**Issue by pheuer** [PlasmaPy/SULI-2021#1]
I'm giving a tutorial on PlasmaPy (especially but not exclusively the HEDP diagnostics) in April, and I want to put together a repository of a few notebooks exhibiting: 
- Basic astropy units, plasmapy Formulary functions
- Thomson scattering
- Charged particle radiography 

This is very similar to the PlasmaPy/SULI-2021 repository. I could just copy that repo and make another one, but what if we changed the name of that into PlasmaPy/BinderDemo or something similar, with the intention of re-using it for different events? This would *not* be a duplication of the docs: there could be some explanation in the notebooks but the emphasis would be on allowing users to actually run the code and make changes themselves.

If too many notebooks are made, we can organize them in folders. And we can have a folder for workshop-specific notebooks in case we want to do something unique for a future workshop? 

Paging @namurphy

**Comment by namurphy** [comment]
Yeah, I think a more general tutorial repository would be great!  We could then either use that repo directly, or fork it for specific tutorials.  It'd probably be worth considering what other packages do, like SunPy and Astropy. We do have an old repo called [PlasmaPy-Tutorials](https://github.com/PlasmaPy/PlasmaPy-Tutorials) which we might be able to use, though we could change it into something else.   

There's also the possibility of using google collab instead of binder, or perhaps another online Python thing.  I've been finding that binder can take a while to spin up an environment, and last I checked, there isn't any persistence between sessions unless someone downloads their notebook.

**Comment by pheuer** [comment]
Astropy has a [good tutorial repo](https://github.com/astropy/astropy-tutorials). SunPy [has one too](https://github.com/sunpy/tutorial-notebooks), but it seems out of date.

I think the repo could be platform-agnostic between binder and collab. Binder has the benefit of not requiring an account (?) so it might make more sense for a quick demo, whereas collab might work better for something like the summer school?

**Comment by pheuer** [comment]
The PlasmaPy-Tutorials repo looks like it was intended to be more tutorials for contributors? 

I'd propose we create a new repo PlasmaPy-Demos, or rename this SULI-2021 repo and use this as the start of that new repo?
